addon/components/infinity-loader.js

```diff
@@ -14,8 +14,21 @@ export default Ember.Component.extend({
   developmentMode: false,
   scrollable: null,
 
+  didRender() {
+    this._super(...arguments);
+    if(Ember.VERSION >= '1.13.0') {
+      this._setup();
+    }
+  },
+
   didInsertElement() {
     this._super(...arguments);
+    if(Ember.VERSION < '1.13.0') {
```

Comment:
I'd be careful about using a semver based approach. Some Ember version numbers [violate semver](https://github.com/emberjs/ember.js/issues/11256). It would be good to add some test cases to [ember-version-is](https://github.com/hhff/ember-version-is) that represent these semver-violating strings.